lib/config_service_placeholders.d.ts

```diff
@@ -308,7 +308,6 @@ export abstract class ConfigurationServicePlaceholders {
   redshiftserverless?: AWS.RedshiftServerless.Types.ClientConfiguration;
   rolesanywhere?: AWS.RolesAnywhere.Types.ClientConfiguration;
   licensemanagerusersubscriptions?: AWS.LicenseManagerUserSubscriptions.Types.ClientConfiguration;
-  backupstorage?: AWS.BackupStorage.Types.ClientConfiguration;
```

Comment:
this line will also probably need removal: [backupstorage apiVersion](https://github.com/aws/aws-sdk-js/blob/6bf0b7d0aa786d7f7209fed369ada0da8417db1f/lib/config_service_placeholders.d.ts#L697)

Reply:
Removed in [`0f6a6fc` (#4647)](https://github.com/aws/aws-sdk-js/pull/4647/commits/0f6a6fc0b5441b86e7f0ff830fe89d14ef2051b2)